google/resource_compute_firewall.go

```diff
@@ -226,13 +333,21 @@ func resourceComputeFirewallDelete(d *schema.ResourceData, meta interface{}) err
 	}
 
 	// Delete the firewall
-	op, err := config.clientCompute.Firewalls.Delete(
-		project, d.Id()).Do()
-	if err != nil {
-		return fmt.Errorf("Error deleting firewall: %s", err)
+	var op interface{}
+	switch computeApiVersion {
```

Comment:
I'm honestly not sure; for now, for completeness for now, I'd like to perform all operations at the resource's API version; we'll be able to bring deletes of versioned resources either down to `v1` or up to `v0beta` every time later, unless we adopt a multiversioned service generated by #240 which makes this mostly a non-issue.